docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,330 @@ sidebar_position: 2
 
 # Quick Start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quik start, you will see a sample application that you deploy and use to
+play around.  As part of this exercise, a backing service and an application is
+required.  The backing service is a PostgreSQL database and application is a
+[Spring Boot REST API server][petclinic].
 
-## K8S
+## Installing Crunchy PostgreSQL for Kubernetes Opeator
```

Comment:
Let's have a quick summary of all pre-requisites and all the steps that the reader will achieve.